### PR TITLE
Adding items display and popup onclick

### PR DESCRIPTION
**Pull request by Talentless-E**
# Summary

---

Reworked minimized version of card
Add popup card by click on minimized card
**Screenshot in case of UI changes**

# Test plan

---

List of steps to manually test introduced functionality:

- Log in
- Go to <http://localhost:3000>
- You should see everything works fine

# Checklist

---

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests for new functionality
- [x] My pull request is shorter than 500 lines

# Screenshots

---
Previous and updated versions of minimized cards

| Previous          | Current          |
| ----------------- | ---------------- |
| image | image |

### CODE REVIEW

**Review by deniserdogan — APPROVED** [review]

File: components/MinimizedCard/MinimizedCard.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    colors: PokemonTypeColor,
+  }
+  const handleClick = id => {
+    pokemonid(id)
+    setActive(true)
+  }
+  return (
+    <Container
+      backgroundColor={typeColor.colors[bgColor]}
+      onClick={() => handleClick(id)}
```
Comment (MaksimVdovin) [inline components/MinimizedCard/MinimizedCard.jsx:17]: Look, you added id in 17 line but function looks like
() => handleClick(id)

What do u think? :)

in your idea it have to be like that
(id)=> handleClick(id)

But onClick event will give u "event" and u will need to take id in handleClick function like
 handleClick = (event) => {...

so, u your case - right option will be like 
   onClick={handleClick}

and function will be like 

 const handleClick = () => {
    pokemonid(id) (it find in scope at this component)
    setActive(true)
  }

File: components/MinimizedCard/MinimizedCard.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    >
+      <ImgContainer
+        style={{ borderBottomLeftRadius: 0, borderBottomRightRadius: 0 }}
+      >
+        <Text
+          color={typeColor.colors[bgColor]}
+          textAlign="right"
+          fontSize="12px"
```
Comment (MaksimVdovin) [inline components/MinimizedCard/MinimizedCard.jsx:25]: finnaly, if u will use default options in styled component (with your idea css inline)

like

export const Text = styled.h4(
  ({ fontSize = '12px', textAlign='right' ... }) => css`
   font: $fontSize;
   ...
  `
)

so, u will not need to use them every time in jsx-components

File: components/MinimizedCard/styles.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    padding: 1px 1px 10px;
+    cursor: pointer;
+    border-radius: 10px;
+  `
+)
+
+export const ImgContainer = styled.div(
+  () => css`
+    display: flex;
+    width: 99%;
```
Comment (MaksimVdovin) [inline components/MinimizedCard/styles.js:21]: what idea to make 99%?)

File: components/PokemonCard/styles.js [hunk 4]
```diff
@@ -1,32 +1,52 @@
...
+  width: 90%;
+  gap: 16px;
+  margin: 24px auto;
+`
+export const BottomContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 16px;
+  background-color: #fff;
```
Comment (MaksimVdovin) [inline components/PokemonCard/styles.js:29]: u can take that from Theme

**Review by nadezhdakharchuk — APPROVED** [review]

**Review by mat1x228 — APPROVED** [review]

**Review by mat1x228 — APPROVED** [review]

**Pull request merged**